data/mods/Magiclysm/monsters/Orcs.json

```diff
@@ -0,0 +1,100 @@
+[
+  {
+    "id": "mon_orc_warrior",
+    "type": "MONSTER",
+    "name": { "str": "Orc Warrior" },
+    "description": "This green brute is a orc warrior, it stares at you menacingly, club in hand.",
+    "default_faction": "orcs",
+    "bodytype": "human",
+    "species": [ "ORCS" ],
+    "volume": "60 L",
+    "weight": "70 kg",
+    "hp": 82,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "o",
+    "color": "green",
+    "aggression": 200,
```

Comment:
200? 100 is about the max needed, as far as I am aware, and that's for berserk things like zombies.